chat-client.ts

```diff
@@ -1,9 +1,11 @@
 import { ChatClientOptions } from "./types";
 
+const WEBSOCKET_URL = "wss://api.commonbase.com/chats";
```

Comment:
Should we put this in this file: https://github.com/commonbaseapp/sdk/blob/main/constants.ts